lume.yml

```diff
@@ -36,14 +36,19 @@ steps:
     envs:
       END2END_TEST: True
       GUNICORN_WORKERS: 1
-      TASK_REPOSITORY_TYPE: sqlite # declared already in pytest.ini
-      SQL_SERVER: sqlite # declared already in pytest.ini
-      SQL_DATABASE: taskmanager_test.db # declared already in pytest.ini
-    setup: docker-compose up -d
+      EVENT_MANAGER_TYPE: rabbitmq
+      TASK_REPOSITORY_TYPE: mysql # declared already in pytest.ini
+      SQL_SERVER: mysql # declared already in pytest.ini
+      SQL_DATABASE: taskmanager
+      MYSQL_DATABASE: taskmanager
+      MYSQL_USER: root
+      MYSQL_PASSWORD: root
+      MYSQL_HOST: localhost
+      MYSQL_PORT: 3307
+    setup: docker-compose up -d && sleep 10
```

Comment:
I had to add this sleep, otherwise it always fails the first time it's launched.

Reply:
Use lume `wait_seconds` instead.